delta/html.py

```diff
@@ -621,6 +621,27 @@ def id_attribute(root, op):
     })
 
 
+def valid_xml_char_ordinal(c):
+    codepoint = ord(c)
+    # conditions ordered by presumed frequency
+    return (
+        codepoint in (0x9, 0xA, 0xD) or
+        0x20 <= codepoint <= 0xD7FF or
+        0xE000 <= codepoint <= 0xFFFD or
+        0x10000 <= codepoint <= 0x10FFFF
+        )
+
+
+def append_text(_to, text):
+    try:
+        if _to:
+            _to += text
+        else:
+            _to = text
```

Comment:
Not sure about `_to += text` but `_to = text` will not update `last.tail`. It will only update local variable `_to`. Same applies to `root` variable.